pom.xml

```diff
@@ -106,55 +106,55 @@
             </resource>
         </resources>
         <plugins>
-            <plugin>
```

Comment:
why commenting that ?

Reply:
It sneaked in, I forgot I did that while playing with generate files :( should be fixed now